package.json

```diff
@@ -3,12 +3,14 @@
   "version": "2.0.13",
   "description": "Midi parser & player engine for browser or Node. Works well with single or multitrack MIDI files.",
   "main": "build/index.js",
+  "browser": "build/index.browser.js",
   "scripts": {
     "pretest": "npm run build",
     "test": "mocha",
-    "build": "mkdir -p build && rollup -c && npm run docs",
+    "build": "rollup -c && npm run docs",
     "watch": "watch 'npm run build' src",
     "docs": "jsdoc src README.md -d ./docs -t ./node_modules/minami",
+    "prepare": "npm run build",
```

Comment:
This is necessary to generate the bundle before installation since the build directory is now being excluded from vcs. For example when installing via git:

```
    "midi-player-js": "git+https://github.com/acmiyaguchi/MidiPlayerJS.git#browser",
```